### PR TITLE
gas-schedule numbers updates.md

### PR DESCRIPTION
**Pull request by cryptomolot**
taken numbers from today transaction.rs and storage_gas.md

### CODE REVIEW

**Comment by saharct** [comment]
@msmouse @vgao1996 can you help update the storage gas section to take into account [AIP-38 - Deprecate storage gas curves](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-38.md)?

File: developer-docs-site/docs/concepts/base-gas.md [hunk 1]
```diff
@@ -276,7 +276,7 @@ Hence, in terms of octas, initial mainnet gas costs can be estimated as follows
 | Operation       | Operation | Minimum octas |
 |-----------------|-----------|---------------|
 | Per-item read   | $r_i$     | 3000          |
-| Per-item create | $c_i$     | 50,000        |
+| Per-item create | $c_i$     | 3000          |
```
Comment (davidiw) [inline developer-docs-site/docs/concepts/base-gas.md:279]:
```suggestion
| Per-item create | $c_i$     | 50,000          |
```

**Review by davidiw — APPROVED** [review]
looks good but the per-item create wasn't modified, still at 50K https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-gas-schedule/src/gas_schedule/transaction.rs#L145

**Comment by davidiw** [comment]
Thanks for these changes, it would be great if you could make a single PR that encompasses all the changes to a single doc. I'll land these ones as is.

**Pull request merged**